packages/storybook/stories/components/Textarea.stories.tsx

```diff
@@ -0,0 +1,183 @@
+/* @license CC0-1.0 */
+
+import type { Meta, StoryObj } from '@storybook/react';
+import { Textarea, TextareaProps } from '@utrecht/component-library-react/src/css-module/Textarea';
+import readme from '@utrecht/components/textarea/README.md?raw';
+import tokensDefinition from '@utrecht/components/textarea/tokens.json';
+import tokens from '@utrecht/design-tokens/dist/index.json';
+import clsx from 'clsx';
+import React from 'react';
+import { designTokenStory } from './util';
+
+interface TextareaStoryProps extends TextareaProps {
+  focus?: boolean;
+  focusVisible?: boolean;
+  hover?: boolean;
+  children?: any;
+}
+
+const arabicDecorator = (Story) => (
+  <div dir="rtl" lang="ar">
+    {Story()}
+  </div>
+);
+
+const TextareaStory = ({ children, focus, focusVisible, hover, ...args }: TextareaStoryProps) => {
+  const classNames = {
+    'utrecht-textarea--focus': focus,
+    'utrecht-textarea--focus-visible': focusVisible,
+    'utrecht-textarea--hover': hover,
+  };
+
+  return <Textarea className={clsx(classNames)} defaultValue={children} {...args} />;
+};
+
+const meta = {
+  title: 'CSS Component/Textarea',
+  id: 'css-textarea',
+  component: TextareaStory,
+  argTypes: {
+    disabled: {
+      description: 'Disabled',
+      control: 'boolean',
+    },
+    invalid: {
+      description: 'Invalid',
+      control: 'boolean',
+    },
+    readOnly: {
+      description: 'Read-only',
+      control: 'boolean',
+    },
+    required: {
+      description: 'Required',
+      control: 'boolean',
+    },
+    focus: {
+      description: 'Focus',
+      control: 'boolean',
+    },
+    focusVisible: {
+      description: 'focusVisible',
+      control: 'boolean',
+    },
+    spellCheck: {
+      description: 'Spellcheck',
+      control: { type: 'select' },
+      options: ['', 'false', 'true'],
+    },
+  },
+  args: {
+    dir: 'auto',
+    disabled: false,
+    focus: false,
+    focusVisible: false,
+    hover: false,
+    invalid: false,
+    readOnly: false,
+    required: false,
+    children: 'The Quick Brown Fox Jumps Over The Lazy Dog',
+  },
+  tags: ['autodocs'],
+  parameters: {
+    status: {
+      type: 'WORK IN PROGRESS',
+    },
+    tokensPrefix: 'utrecht-textarea',
+    tokens,
+    tokensDefinition,
+    docs: {
+      description: {
+        component: readme,
+      },
+    },
+  },
+} satisfies Meta<typeof Textarea>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  args: {
+    children: 'The Quick Brown Fox Jumps Over The Lazy Dog',
+  },
+  parameters: {
+    docs: {
+      description: {
+        story: `Styling via the \`.utrecht-textarea\` class name.`,
+      },
+    },
+  },
+};
```

Comment:
De standaard Placeholder story is verdwenen.

## Main branch

<img width="259" alt="Screenshot 2023-09-05 at 08 08 27" src="https://github.com/nl-design-system/utrecht/assets/30694/a537a9c4-fd2b-4c99-b9d4-2ffe5edb51bc">

## Deze branch

<img width="333" alt="Screenshot 2023-09-05 at 08 08 32" src="https://github.com/nl-design-system/utrecht/assets/30694/18d52be9-9822-419a-8a96-6813bd743f55">